### ISSUE TITLE
[Aging Content - Full-width Banner] A11y Audit for  Design:  Full-width Banner Alerts Date limits and notifications #15495

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#16014]
**1. What is this request for?**

- Content Editors will see a new date-limit widget and text when they are creating Full-width banners so they are aware the banner will have an expiration date, and what that date is.
- Three different emails will be sent on different cadences to alert the editor that:
  - the banner's expiration date is approaching, at which time it will be archived and no longer displayed
  - the banner will be expiring 'today'
  - the banner has been archived and no longer displays

**2. What is the primary function and/or purpose of the product?**

- The date widget is to provide a means of time-boxing the amount of time a new full-width banner will be displayed to Veterans and their caregivers so that we ensure only timely, current information is displayed to them. The date widget will default to 7 days.
- The emails are to provide prior warning to the editor that their banner will be expiring/the expiration date has arrived/the banner has been archived and no longer displays.

**3. Are there any known accessibility issues that have not yet been addressed? If so, please add GitHub links to those issues below.**
  
**4. Are there any specific testing requirements that should be noted?**
- 
  
**5. Please provide anything needed to access and review the product.**
  
- This is the form where we'll be adding the date expiration stuff: [https://prod.cms.va.gov/node/add/banner]
- This [Figma file](https://www.figma.com/file/n2F9Y0QhjNoeNyQ4zv2VER/Full-Width-Banner-Alerts?type=design&node-id=0%3A1&mode=design&t=dWmDMYhlqBe68kUL-1) that shows content type updates and email template
  
**6. When does the product need to launch?**

> All requests should be submitted at least 1 sprint in advance so that it can be worked into the next sprint.  For requests needed sooner, please notify Laura Flannery in the cms-support channel.
